packages/app/source/client/controllers/todos-controller.js

```diff
@@ -6,13 +6,17 @@ Space.Object.extend(Todos, 'TodosController', {
   ],
 
   dependencies: {
-    configuration: 'configuration'
+    configuration: 'configuration',
+    todosStore: 'Todos.TodosStore',
```

Comment:
Let's wait to hear @DominikGuzei 's take on this. For me it seems better to do this in the component, as the only real reason we need a controller is that the UI event needs to be translated to a command and should be a router type object. This goes for Donations too